WaarpCommon/src/main/java/org/waarp/common/logging/PrintOnlyWarningLogbackStatusListener.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of Waarp Project (named also Waarp or GG).
+ *
+ *  Copyright (c) 2019, Waarp SAS, and individual contributors by the @author
+ *  tags. See the COPYRIGHT.txt in the distribution for a full listing of
+ * individual contributors.
+ *
+ *  All Waarp Project is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * Waarp is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with
+ * Waarp . If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.waarp.common.logging;
+
+import ch.qos.logback.core.status.OnConsoleStatusListener;
+import ch.qos.logback.core.status.Status;
+
+import java.util.List;
+
+public class PrintOnlyWarningLogbackStatusListener
+    extends OnConsoleStatusListener {
+
+  private static final int LOG_LEVEL = Status.WARN;
+
+  @Override
+  public void addStatusEvent(Status status) {
+    if (status.getLevel() == LOG_LEVEL) {
```

Comment:
If I understand it well, this class allows to filter the messages displayed by logback when it starts. If so:

* this test only prints warning, but skips error-level statuses. Mybe replace `==` by `>=`
* Would it be possible to make this the default ? I think the filtered version matches "normal" operations, and if we want to really have all the output, we can override it in the logback XML conf (with `debug="true"` or by adding `OnConsoleStatusListener`)

If it is not the purpose of this class, please diregard my comment !

Reply:
This was from an example on website, but you are definitively right on `>=`.

Sadly, this could not be done programmatically. One has to add to the XML configuration of Logback the following line:

`<statusListener class="org.waarp.common.logging.PrintOnlyWarningLogbackStatusListener" />`

Reply:
Done

Reply:
:+1: